doc/lua_api.md

```diff
@@ -8408,6 +8410,9 @@ child will follow movement and rotation of that bone.
       * `shadows` is a table that controls ambient shadows
         * `intensity` sets the intensity of the shadows from 0 (no shadows, default) to 1 (blackness)
             * This value has no effect on clients who have the "Dynamic Shadows" shader disabled.
+        * `tint` tints the shadows with the provided color, with RGB values ranging from 0 to 1.
```

Comment:
uhh wait. is this a float rgb vector?
if so, why? we use a [0, 255] range everywhere else

Reply:
Crikey, I must have been really tired when I wrote this... It's just a normal color using normal int values.